engine/src/vs_random.h

```diff
@@ -31,8 +31,6 @@
  *  Any feedback is very welcome.
  *  http://www.math.keio.ac.jp/matumoto/emt.html
  *  email: matumoto@math.keio.ac.jp
- */
-// NO HEADER GUARD
```

Comment:
we need to keep the `NO HEADER GUARD` comment

Reply:
Fixed